### PR TITLE
added exception for wait_time_median.get_value() returning None

### LINKED ISSUE #455: Raise exception for get_value in resolve_median
_Originally posted by @youngj in https://github.com/trynmaps/metrics-mvp/pull/372_

### PR DESCRIPTION
**Pull request by Brian-Lee**
Fixes #455

## Link to demo, if any
https://metrics-mvp-brians-fork.herokuapp.com/

This is my first pull request. I am not certain I fixed the issue #455 as it is not trivial for me to generate and observe the issue behavior. However, I think it might fix it, and I don't think it should hurt.

### CODE REVIEW

**Comment by Brian-Lee** [comment]
Good suggestion. I'll get on it.

**Comment by Brian-Lee** [comment]
Ok. I created the local variable.

File: backend/models/schema.py [hunk 1]
```diff
@@ -19,7 +19,10 @@ def resolve_median(parent, info):
         wait_time_median = wait_times.get_cached_wait_times(parent['route_metrics'].agency_id, first_date, "median", parent["start_time"], parent["end_time"])
         if wait_time_median is None:
             raise Exception(f"There is no cached median for start_stop_id {parent['start_stop_id']} at times {parent['start_time'], parent['end_time']}.")
-        return wait_time_median.get_value(parent["route_metrics"].route_id, parent["direction_id"], parent["start_stop_id"])
+		wait_time_median_value = wait_time_median.get_value(parent["route_metrics"].route_id, parent["direction_id"], parent["start_stop_id"])
```
Comment (youngj) [inline backend/models/schema.py]: Looks like this line uses tabs instead of spaces

Comment (Brian-Lee) [reply backend/models/schema.py]: Ooops. Ok. I think I fixed the indentationsnow.

**Pull request merged**